### PR TITLE
Update global tool and resolve build warnings.

### PR DESCRIPTION
**Pull request by JunTaoLuo**
Addresses https://github.com/grpc/grpc-dotnet/issues/360 and https://github.com/grpc/grpc-dotnet/issues/506. Also resolves a few pack warnings.

### CODE REVIEW

File: src/dotnet-grpc/runtimeconfig.template.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "rollForwardOnNoCandidateFx": 2
```
Comment (JamesNK) [inline src/dotnet-grpc/runtimeconfig.template.json]: Can you add a comment of what this does

**Comment by JunTaoLuo** [comment]
They show up when you pack, not during build.

File: src/dotnet-grpc/runtimeconfig.template.json (hunk 1 above)
Comment (JunTaoLuo) [reply src/dotnet-grpc/runtimeconfig.template.json]: Not in this file? I'll add it in the csproj.

File: Directory.Build.targets [hunk 2]
```diff
@@ -23,6 +24,8 @@
 
   <!-- IsGrpcPublishedPackage is set in csproj so related config must be in targets file -->
   <ItemGroup Condition="'$(IsGrpcPublishedPackage)' == 'true'">
+    <None Include="$(PackageIconFullPath)" Pack="true" PackagePath="\"/>
```
Comment (JamesNK) [inline Directory.Build.targets]: What is this for?

Comment (JunTaoLuo) [reply Directory.Build.targets]: The docs recommend this: https://docs.microsoft.com/en-us/nuget/reference/msbuild-targets#packing-an-icon-image-file

File: src/Grpc.AspNetCore/Grpc.AspNetCore.csproj [hunk 3]
```diff
@@ -13,6 +13,12 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <!-- Avoid NU5128 warning. TFM in dependencies group of the nuspec must match lib/ref content. -->
+    <None Include="lib\**\*">
```
Comment (JamesNK) [inline src/Grpc.AspNetCore/Grpc.AspNetCore.csproj]: Does this include a DLL in the package? I explicitly excluded this with `IncludeBuildOutput` on line 11.

Comment (JunTaoLuo) [reply src/Grpc.AspNetCore/Grpc.AspNetCore.csproj]: No it does not, it includes just the dummy file.

image

**Pull request merged**